app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This layout is fairly simple, you can use `LinearLayout` instead (same for the other fragments)

Reply:
I will keep this in mind for future components. However, should I change this considering we will modify the layout?

Reply:
Nah, it's ok 😁